Update overview-and-concepts.mdx (#5840)
EnterpriseDB/docs: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx

## [EDB Postgres AI Console](/edb-postgres-ai/console)
The Console is the gateway to EDB Postgres AI. Starting with a single pane of glass onto all your EDB Postgres AI operations, Console is there to manage your database landscape. It also lets you create and manage your databases, and provides a view of your databases' health and performance. Add to that the ability to bring up Lakehouse Analytics and AI/ML capabilities, and you have the power of EDB Postgres AI at your fingertips.
* **[EDB Postgres AI Estate](/edb-postgres-ai/console/estate)**
  * EDB Postgres AI Cloud Service databases are automatically managed on the Console. The Estate can also monitor the health of AWS RDS and S3 resources and using the Agent, on-premise databases are shown and managed in the same view.
* **[EDB Postgres AI Agent](/edb-postgres-ai/console/agent)**

>   * Include on premise databases under the Console's single pane of glass view with the Agent. The Agent is part of enabling EDB Postgres AI an unprecedented view of diverse deployments.

Reviewer: It would read better. 

```suggestion
  * Include on premise databases under the Console's single pane of glass view with the Agent. The Agent is part of giving EDB Postgres AI an unprecedented view of diverse deployments.
```

## [EDB Postgres AI Cloud Service](/edb-postgres-ai/cloud-service)
Not just databases, but driven by databases, Cloud Service provides a global platform for delivering new elements of EDB Postgres AI efficiently and effectively.
* **[Cloud Service Hosted](/edb-postgres-ai/cloud-service/hosted)**
  * The Cloud Service Hosted environment is where you can experience the full power of EDB Postgres AI, with all the elements of the platform available to you.
* **[Cloud Service Managed](/edb-postgres-ai/cloud-service/managed)**
  * Bring your own cloud account to EDB and let the Managed Service environment take care of deploying and managing your Postgres databases in your own cloud provider account

## [EDB Postgres AI Databases](/edb-postgres-ai/databases)

> All of EDB's database expertise can be found in EDB Postgres Advanced Server and EDB Postgres Extended Server. Oracle compatibility, TDE (Transparent Data Encryption) and more. They provide the data fabric on which EDB Postgres AI operates. Combined with EDB Postgres Distributed, they can also provide a high availability environment for your data. 

Reviewer 2: ```suggestion
All of EDB's database expertise can be found in EDB Postgres Advanced Server and EDB Postgres Extended Server, including Oracle compatibility, transparent data encryption (TDE), and more. They provide the data fabric on which EDB Postgres AI operates. Combined with EDB Postgres Distributed, they can also provide a high-availability environment for your data. 
```

All of these components are available on the EDB Postgres AI Cloud Service, and managed through the EDB Postgres AI Console.
* **[EDB Postgres Advanced Server](/edb-postgres-ai/databases/epas)**

>   * EDB Postgres Advanced Server provides mission critical capabilities for your data along with Oracle compatibility.

Reviewer 2: ```suggestion
  * EDB Postgres Advanced Server provides mission-critical capabilities for your data along with Oracle compatibility.
```

Filtering out the data noise and revealing insights and value, Lakehouse analytics brings both structured relational data in Postgres and unstructured data in object storage together for exploration.

* **[Lakehouse nodes](/edb-postgres-ai/analytics/lakehouse)**

>   * At the heart of Analytics is a custom built object storage for your data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring cold data in ready for analysis. 

Reviewer 2: ```suggestion
  * At the heart of Analytics is custom-built object storage for your data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring cold data in, ready for analysis. 
```

## [EDB Postgres AI AI/ML](/edb-postgres-ai/ai-ml)

> * Postgres has proven its capability as a flexible data environment, and with Vector data as the core of generative AI, is already infused into EDB Postgres AI providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres "pgai" extension.

Reviewer 2: ```suggestion
* Postgres has proven its capability as a flexible data environment. With vector data as the core of generative AI, ir's already infused into EDB Postgres AI, providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres pgai extension.
```

Reviewer 2: Please check if this is correct. It was hard to follow.

Reviewer: ```suggestion
* Postgres has proven its capability as a flexible data environment and EDB Postgres AI is already infused with Vector data support, enabling generative AI and providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres "pgai" extension.
```
